.licenserc.yaml

```diff
@@ -0,0 +1,24 @@
+header:
```

Comment:
Consider adding the ignored paths to the operator workflows if the files are common across repositories